docs/api/secret.md

```diff
@@ -198,6 +199,19 @@ Updates parameters of an existing secret.
     }
     ```
 
```

Comment:
This looks OK.  Since you are giving instructions for how to update from the command-line,
I suggest that you extend this example slightly, by using the `base64` command line utility to encode the secret.